lib/elixir/lib/kernel.ex

```diff
@@ -3334,6 +3334,24 @@ defmodule Kernel do
       end
       #=> warning: the underscored variable "_bar" is used after being set
 
+  ## rescue/catch/after
+
+  Function bodies support `rescue`, `catch` and `after` as `SpecialForms.try/1`
+  does. The following two functions are equivalent:
+
+      def format(value) do
+        try do
+          format!(value)
+        catch
+          :exit, reason -> {:error, reason}
+        end
+      end
+
+      def format(value) do
+        format!(value)
+      catch
+        :exit, reason -> {:error, reason}
+      end
   """
```

Comment:
We usually leave an empty line between the last line of a code sample and the closing `"""` :)